### ISSUE TITLE
About the batch size and learning rate

### ISSUE THREAD

**Issue by hytu99** [ruotianluo/ImageCaptioning.pytorch#128]
Thanks for the excellent codebase! I find that the default batch size is 10 and the default initial learning rate is 0.0005 for the updown model. But in the original paper, the batch size is 100 and the initial learning rate is 0.01. 

I wonder why you choosed a smaller batch size and a smaller learning rate? Can I use a larger batch size and a larger learning rate to make the training faster? Will it harm the performance of the model?
